substrate/frame/collective/src/lib.rs

```diff
@@ -350,6 +357,10 @@ pub mod pallet {
 		WrongProposalLength,
 		/// Prime account is not a member
 		PrimeAccountNotMember,
+		/// There can only be a maximum of `MaxMembers` votes per proposal.
```

Comment:
```suggestion
		/// There can only be a maximum of [`MaxMembers`] votes per proposal.
```

Reply:
This does not appear to be resolved?

Reply:
the square brackets introduce the following warning in `cargo doc --no-deps -p pallet-collective` 
![image](https://github.com/paritytech/polkadot-sdk/assets/56828990/0e66582e-2210-4e06-8c63-d3999f73c906)

Reply:
This should work
```suggestion
		/// There can only be a maximum of [`T::MaxMembers`] votes per proposal.
```

Reply:
![image](https://github.com/paritytech/polkadot-sdk/assets/56828990/801fe2b0-b0e5-41f5-b500-78a20b68226f)
Nope, not even `<T as Config<I>>`.

Reply:
https://github.com/paritytech/polkadot-sdk/blob/a70617124b4c0c730016a210e2b8b343c829d784/substrate/frame/alliance/src/lib.rs#L349 
It is done this way in other pallets as well.